src/main/java/uk/gov/ons/ctp/response/sample/service/SampleSummaryActivationService.java

```diff
@@ -0,0 +1,135 @@
+    collectionExerciseStatus.setSuccessful(successful);
+    collectionExerciseStatus.setEvent(SampleSummaryStatusDTO.Event.ENRICHED);
+    sampleSummaryActivationStatusPublisher.sendSampleSummaryActivation(collectionExerciseStatus);
+  }
+
+  public void sendDistributeStatusToCollectionExercise(
+      UUID collectionExerciseId, boolean successful) {
+    SampleSummaryStatusDTO collectionExerciseStatus = new SampleSummaryStatusDTO();
+    collectionExerciseStatus.setCollectionExerciseId(collectionExerciseId);
+    collectionExerciseStatus.setSuccessful(successful);
+    collectionExerciseStatus.setEvent(SampleSummaryStatusDTO.Event.DISTRIBUTED);
+    sampleSummaryActivationStatusPublisher.sendSampleSummaryActivation(collectionExerciseStatus);
+  }
+
+  /**
+   * Executes the first step of the activation process. This will validate all the sample unit data
+   * and enrich the sample unit by gathering any extra data required for the second step of the
+   * activation process.
+   *
+   * @param sampleSummaryActivation an object containing all the details required for activation
+   */
+  private void validateAndEnrich(SampleSummaryActivationDTO sampleSummaryActivation)
+      throws SampleSummaryActivationException {
+    UUID sampleSummaryId = sampleSummaryActivation.getSampleSummaryId();
+    UUID surveyId = sampleSummaryActivation.getSurveyId();
+    UUID collectionExerciseId = sampleSummaryActivation.getCollectionExerciseId();
+
+    LOG.debug(
+        "about to enrich sample summary",
+        kv("sampleSummaryId", sampleSummaryId),
+        kv("surveyId", surveyId),
+        kv("collectionExerciseId", collectionExerciseId));
+
+    try {
+      boolean validated =
+          sampleSummaryEnrichmentService.enrich(surveyId, sampleSummaryId, collectionExerciseId);
+      LOG.debug(
+          "Enriched sample summary",
+          kv("sampleSummaryId", sampleSummaryId),
+          kv("surveyId", surveyId),
+          kv("collectionExerciseId", collectionExerciseId),
+          kv("validated", validated));
+      if (validated) {
+        LOG.info("Sample summary successfully enriched", kv("sampleSummaryId", sampleSummaryId));
+      } else {
+        LOG.error("Validation and enrichment failed", kv("sampleSummaryId", sampleSummaryId));
+        throw new SampleSummaryActivationException();
+      }
+    } catch (UnknownSampleSummaryException e) {
+      LOG.error("unknown sample summary id", kv("sampleSummaryId", sampleSummaryId), e);
+      sendEnrichStatusToCollectionExercise(collectionExerciseId, false);
+      throw new SampleSummaryActivationException();
+    } catch (SampleSummaryActivationException e) {
+      LOG.error(
+          "Something went wrong activating sample summary",
+          kv("sampleSummaryId", sampleSummaryId),
+          e);
+      sendEnrichStatusToCollectionExercise(collectionExerciseId, false);
+      throw new SampleSummaryActivationException();
```

Comment:
rather than throwing a new SampleSummaryActivationException you could rethrow the caught one, i.e. 

```
  throw e;
```

It'll keep the stack trace then